packages/client/src/admin/Secrets/Secrets.tsx

```diff
@@ -66,21 +66,25 @@ function _Secrets({
   const history = useHistory();
   const location = useLocation();
   const querystring = new URLSearchParams(location.search);
+  const compareString = (a: string, b: string) => (a ? a.localeCompare(b) : -1);
```

Comment:
I think can simply write as bleow:

```suggestion
  const compareString = (a: string, b: string) => a.localeCompare(b);
```